src/main/java/com/carhouse/model/stub/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
```

Comment:
looks like this package is redundant and should be deleted

Reply:
Checkstyle says what I have to have package-info.java in every package, so I can't remove it